ddtrace/opentrace/scope.py

```diff
@@ -0,0 +1,10 @@
+from opentracing import Scope as OpenTracingScope
+
+
+class Scope(OpenTracingScope):
+    """"""
```

Comment:
👍 let's keep them empty in the skeleton, and let's write them every time we implement something